src/channel_server/key_map_key.hpp

```diff
@@ -73,10 +75,12 @@ namespace vana {
 		m = 50,
 		comma = 51,
 		dot = 52,
-		// ??
+		slash = 53,
+		rightshift = 54, // Maps to 'shift' automatically
```

Comment:
This should be `right_shift`. I would also recommend changing `shift = 42` to `left_shift = 42` as well.